api/harbor/swagger.yaml

```diff
@@ -6236,6 +6236,9 @@ definitions:
       description:
         type: string
         description: The description of robot account
+      expires_at:
+        type: integer
+        description: The expiration of robot account (in seconds)
```

Comment:
This description is misleading.

I think you mean the number of seconds since 1970-01-01?  I recall it's called epoch time?  Please double check.

Reply:
updated